http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriberModels.scala

```diff
@@ -0,0 +1,215 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package monitor
+
+import io.circe.syntax._
+import io.circe.{Decoder, DecodingFailure, Encoder}
+import org.broadinstitute.dsde.workbench.google2.JsonCodec.traceIdDecoder
+import org.broadinstitute.dsde.workbench.google2.JsonCodec.traceIdEncoder
+import org.broadinstitute.dsde.workbench.leonardo.JsonCodec._
+import org.broadinstitute.dsde.workbench.leonardo.model.google.{ClusterName, ClusterStatus, IP, OperationName}
+import org.broadinstitute.dsde.workbench.leonardo.model.{Cluster, ClusterProjectAndName, DataprocInfo}
+import org.broadinstitute.dsde.workbench.leonardo.monitor.LeoPubsubMessage._
+import org.broadinstitute.dsde.workbench.model.TraceId
+import org.broadinstitute.dsde.workbench.model.google.GcsPath
+
+import scala.util.control.NoStackTrace
+
+sealed trait LeoPubsubMessage {
+  def traceId: Option[TraceId]
+  def messageType: String
+}
+
+object LeoPubsubMessage {
+  final case class StopUpdateMessage(updatedMachineConfig: RuntimeConfig, clusterId: Long, traceId: Option[TraceId])
+      extends LeoPubsubMessage {
+    val messageType = "stopUpdate"
+  }
+
+  case class ClusterTransitionFinishedMessage(clusterFollowupDetails: ClusterFollowupDetails, traceId: Option[TraceId])
+      extends LeoPubsubMessage {
+    val messageType = "transitionFinished"
+  }
+
+  case class CreateCluster(id: Long,
```

Comment:
Do we want to suffix this with message to stay consistent with the other two? Im fine with dropping it from the other two if we want to be less verbose since its already a part of LeoPubsubMessage (which it wasn't when they were originally named), but I think we should stay consistent.

Reply:
I'll get rid of the suffix for the other ones.